### ISSUE TITLE
Does not support assigning a script to a new clip

### ISSUE THREAD

**Issue by mtsanaissi** [sabrogden/Ditto#671]
Would it be possible to use a script in the new clip creation? My use case is that I would like a clip, which I would be able to make sticky / never delete / save to a group, to run a script as simple as get current date in YYYY-MM-DD format and paste the result. I already have the script.
I know I can set a hotkey to activate the script but I'd prefer if it could be a clip.
Thank you for this very helpful piece of software!

**Comment by Chealer** [comment]
@mtsanaissi: Is this still relevant for a current version? If so, can you clarify what you are asking for? Are you asking the possibility to create clip pseudo-entries which do nothing but invoking a script?

**Comment by mtsanaissi** [comment]
Yes @Chealer. Well I don't know if I would use it nowadays because I am settled with another solution for this specific use case (although I still use Ditto!).
In another words, my use case was to be able to create a special kind of clip where the pasted text would be defined by a script.
At the time, the script was a simple one to get current date:
`var date = FormatCurrentTime("%Y-%m-%d");
clip.SetAsciiString(date);
return false;`
Not sure if this is possible with current version, though.

**Comment by Chealer** [comment]
@mtsanaissi: This is an ITS. Making implicit requests is fine, but each ticket is supposed to first focus on describing a problem.

At this time, Ditto is not designed to insert anything other than something copied. If you are looking for ways to insert a current timestamp, see ticket #628.

The current title is both unclear and inappropriate for an issue report. If this request is specifically about inserting "timestamps", please retitle to something like:

> Does not properly support inserting current date/timestamp

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>

**Comment by mtsanaissi** [comment]
You are right, @Chealer. I could not find a better place to request this feature. As this is not a problem in Ditto, I will close the issue. Thank you!
